mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -86,6 +87,21 @@ class LogisticRegressionSuite extends MLTest with DefaultReadWriteTest {
       df
     }
 
+    binaryDatasetWithSmallVar = {
+      val nPoints = 10000
+      val coefficients = Array(-0.57997, 0.912083, -0.371077, -0.819866, 2.688191)
+      val xMean = Array(5.843, 3.057, 3.758, 1.199)
+      val xVariance = Array(0.6856, 0.1899, 3.116, 0.0001)
```

Comment:
Why not increase the xMean to a big value like 100 for the feature with var=0.0001
 ?

Reply:
update xMean to 10.199;
if I update it to 100.199, then all labels sampled from `generateMultinomialLogisticInput` are negative, that is to say, all label=0. Can not train a model in R or MLLib.